### DISCUSSION TITLE
有没有直接批量替换多个智能对象的免费脚本

### DISCUSSION THREAD

**Discussion by here555** [joonaspaakko/Batch-Mockup-Smart-Object-Replacement-photoshop-script#27] [Q&A]
有没有直接批量替换多个智能对象的免费脚本，我没有任何的编程知识，可以直接发做好的脚本帮助我吗

**Comment by joonaspaakko** [comment]
Google Translate suggests you asked:
> Is there any free script to directly replace multiple smart objects in batches? I don’t have any programming knowledge. Can you help me by directly sending the finished script?

This script does exactly that and the script is in this repository if you just download it. Unfortunately, if there's enough of a language barrier that you had to ask this question, I'm not sure if I can help you... You could try to follow the instructions in [the documentation that talks about using the standalone example as a starting point](https://joonaspaakko.gitbook.io/batch-mockup-smart-object-replacement-jsx-photosho/#option-1-use-a-project-template), which makes it way easier to use, since you just replace the placeholder items with your own custom setup. 

One thing you could try is to maybe find some sort of graphic design or Photoshop community with people who speak your language and see if they can steer you in the right direction with this script or anything else. 

Another thing that comes to mind is the website [placeit.net](https://placeit.net/), that doesn't require any scripting knowledge and barely any graphic design knowledge, if any. Of course, you can't use just any mockup there, but they do have a rather large library of mockups... It's not very expensive for a singular output image, but if you want a large amount of mockups and images, I it might be better to use a script like this one.

**Comment by here555** [comment]
我刚用这个软件平台不久，对于这个Change mockupPath to po更改和psd路径更改您可以出个视频吗？因为我实在太笨了(／_＼)以下是我的工作视频

**Reply by joonaspaakko** [reply]
Alright fine; [link to the video](https://www.dropbox.com/scl/fi/zyy3v63za8kte4ssx4b25/Screen-Recording-2023-08-19-at-14.17.23.mov?rlkey=ceaapuatxbd8qg7zetr6qreth&dl=0).
